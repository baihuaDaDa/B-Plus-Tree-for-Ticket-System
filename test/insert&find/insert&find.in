1000
find G
insert CaW 89
insert CaW 88
insert CaW 89
insert BcN 49
find CaW
find BcN
find CaW
find e
insert BcN 49
find CaW
insert CaW 88
find CaW
insert CaW 3
insert CaW 3
insert huE 76
insert huE 48
insert VBV 32
find huE
insert CaW 3
insert CaW 89
insert huE 48
insert ncJ 88
insert Fly 46
insert huE 48
find CaW
insert BJw 21
find CaW
insert CaW 3
insert CaW 89
find VBV
insert FhE 23
insert CaW 89
insert Sib 32
find huE
insert nEU 75
find BcN
find CaW
find f
insert BcN 29
insert CaW 89
insert huE 23
insert FtW 76
insert Dnz 63
insert APf 38
insert BcN 49
find CaW
insert cJa 88
find BcN
insert CaW 89
insert hup 36
insert BcN 29
insert gwu 11
insert BcN 29
insert VBV 32
insert CaW 21
find gwu
find CaW
insert BVn 36
insert huE 23
find CaW
find CaW
insert CaW 51
insert ObA 27
find VBV
insert xpf 78
insert Dnz 63
insert akm 90
find i
find Fly
insert CaW 68
insert Fnm 48
find y
find BJw
insert VBV 32
insert tYZ 31
find q
find xpf
find BcN
find g
find Y
find L
insert CaW 89
find cJa
insert lTm 6
find BcN
insert Fnm 26
insert aar 31
insert QEL 14
find M
insert Fbd 45
find Z
find cJa
insert FtW 76
find FtW
insert gwu 11
insert BcN 30
find CaW
find CaW
insert nAK 33
find p
insert huE 23
insert BcN 42
find huE
insert fpe 85
insert CaW 63
find CaW
find CaW
insert ExP 84
find BVn
find CaW
insert BcN 49
find Fnm
insert BJw 21
insert HkZ 57
find CaW
insert nAK 33
find Fly
insert HXl 31
find I
insert hup 36
insert lWH 85
insert wkz 57
find huE
insert tRQ 66
find FtW
insert CEc 24
find CaW
insert FtW 15
insert BJw 21
insert eWS 26
insert cJa 88
insert Njr 19
find BcN
insert rtf 60
find HkZ
find BcN
insert huE 23
insert ctf 81
insert CaW 3
insert BJw 21
insert vQG 70
find rtf
find CaW
insert BcN 87
insert CaW 88
insert nAK 33
find CaW
find j
find C
insert HkZ 57
find CaW
find v
insert CaW 89
find B
insert CaW 3
insert VBV 24
find Dnz
insert gtT 93
find BcN
insert huT 26
find CaW
find cJa
find y
find huE
insert CaW 3
find A
find Fly
find Fly
find CaW
find CaW
insert Sib 32
insert qpQ 84
find HXl
insert euc 38
insert aPv 24
find g
find huE
insert Fbd 51
insert OSt 16
insert huE 76
find R
find tYZ
insert CaW 88
find L
insert HkZ 57
find cJa
insert FtW 40
insert BcN 30
insert RDK 42
insert Fly 46
insert CMa 92
insert REx 83
insert haL 35
find CaW
insert OEo 47
find BcN
find Dnz
insert UVR 34
find FtW
insert VBV 46
insert lWH 85
find lWH
insert BcN 69
insert IVD 51
find VBV
insert qpQ 84
insert ctf 6
insert tRQ 2
insert cJa 88
insert CaW 15
find CaW
insert lTm 6
insert uoi 79
insert ObA 27
insert hhu 47
insert RDK 42
insert Ohr 85
find HkZ
insert BXP 55
insert CaW 45
find CaW
find REx
insert CaW 3
insert CaW 88
insert lQv 57
insert FtW 76
insert nAK 33
insert euc 38
insert XTU 79
find huE
find lTm
find CaW
find lWH
insert CaW 88
insert YJs 95
find BJw
find BcN
insert ysx 47
find wkz
find O
insert lWH 85
find CaW
insert cWo 40
insert htQ 99
find k
insert BVn 45
insert wuy 5
find huE
find W
find BVn
find w
insert CaW 45
find aar
insert uoi 44
insert CaW 15
insert VBV 40
insert BcN 6
insert qpQ 84
find huE
insert sJn 36
insert uoi 79
insert lWH 44
find uoi
find FhE
insert CaW 93
insert CaW 88
find M
insert FtW 36
find x
insert vQG 70
find huE
insert SRW 33
find CaW
insert uoi 8
insert lWH 23
insert dqX 69
insert vAy 45
insert tVA 97
insert CEc 24
find Fly
insert DSo 48
find lWH
find j
insert sha 93
insert mnO 84
insert huT 26
find BJw
insert gZR 36
find lWH
insert Sib 32
insert VBV 14
insert YWd 38
insert nAK 33
find aPv
insert Eem 67
insert BJw 21
find wuy
insert VoC 59
insert nAK 33
insert CaW 3
insert lWH 23
find Njr
insert VoC 57
insert RDK 42
insert lhF 7
insert gwu 76
insert nAK 33
insert uBW 18
insert HYE 14
insert RDK 65
insert OSt 16
find A
find CaW
find G
insert FtW 76
find BcN
find ncJ
find b
find htQ
insert pao 74
insert myG 88
insert huE 25
find E
insert BXP 55
find VBV
insert tEL 98
find CaW
insert uBW 18
find c
insert Bza 78
insert lUb 23
insert CaW 67
find k
find I
find vQG
insert CaW 88
insert CaW 15
find gwu
insert tRQ 2
insert GwD 78
insert aip 38
insert fkX 14
insert Hwt 52
find huE
find vAy
insert uBW 77
find RDK
find huT
find HkZ
insert euc 24
insert Fly 46
insert Ixg 8
find x
insert FtW 76
insert BcN 6
insert lhF 7
insert ncJ 40
insert jXi 33
insert JGo 1
find Fbd
find fkX
insert VMt 32
insert FtW 40
insert htU 12
insert ctf 98
find uoi
find cJa
insert Xlv 82
find J
insert seg 39
find euc
find CaW
insert CaW 89
insert Sib 32
insert Fly 80
find CaW
insert vQG 14
insert gtT 93
insert PtO 14
find haL
find BcN
find G
insert gwu 76
insert DuH 58
find CaW
insert CaW 79
insert hup 28
find CaW
insert Sib 32
find huT
insert ObA 27
insert Fly 27
insert BPw 29
find Q
insert Fbd 13
insert haL 65
insert ESn 36
insert VBV 14
insert CaW 88
find N
insert ncJ 40
insert Mtv 17
find VoC
insert SBj 73
find c
insert tRG 52
find CaW
insert gwu 11
insert hFH 5
find A
find gtT
insert CaW 88
find GwD
insert FtW 76
insert ubZ 6
find CaW
insert xpf 49
insert Fly 46
find ctf
insert gZR 36
insert uoi 79
insert CaW 89
find R
insert ZRp 28
insert ncJ 88
find nAK
find CaW
insert HkZ 94
insert CaW 3
insert cJa 88
find hup
find huE
insert TMm 64
insert lWH 23
insert ObA 68
insert Sib 32
insert PtO 75
insert dQm 87
insert BcN 49
find huE
find CaW
find huE
insert OSt 43
find lWH
insert fZA 22
find hup
find Fly
insert vQG 9
find ObA
find xpf
insert UzF 63
find rtf
insert huE 91
insert KxJ 86
insert dDt 75
insert HkZ 57
find CaW
find u
insert qWH 8
find BJw
find CaW
insert kpC 2
insert CaW 89
insert VoC 57
insert QYW 31
insert jQB 45
find gtT
insert CaW 89
find BcN
find CEc
insert BcN 49
insert YWd 38
insert CaW 3
find RDK
insert zuG 73
find BJw
find ysx
find myG
insert AMj 92
find FtW
insert cfQ 21
insert BPc 51
insert GpE 20
insert RDK 65
insert gwu 11
find Xlv
insert DSo 47
insert ltU 23
insert zzq 8
insert lWH 23
insert qWH 8
find Y
insert gsf 37
find VBV
insert dxP 41
insert erP 2
insert BFd 26
find f
insert YXg 99
find CaW
insert OGg 63
find CaW
insert rzg 34
insert WWF 4
insert huE 48
find CaW
insert AMj 92
find jQB
insert CaW 88
find htU
insert vQG 70
insert VBV 32
insert FtW 36
insert myG 88
insert Fly 27
find uBW
find Ixg
find cfQ
find gZR
insert eJt 92
find dDt
find a
insert UUL 18
insert gtT 93
insert Cwe 7
insert HkZ 57
insert myG 88
find Fnm
find d
find CaW
insert uBW 52
insert hue 17
insert bfU 89
insert Mtv 99
find gwu
insert twH 22
insert VBV 14
insert hup 36
insert aip 38
insert xLn 59
find Fly
insert EGA 53
insert FtW 7
insert epK 70
find BcN
insert HkZ 57
find F
find HkZ
insert Fly 46
find VBV
find lWH
find huE
find CaW
find S
find O
find Fly
insert GZa 42
insert cWo 40
find UzF
find p
insert ZiY 74
insert haL 65
find CaW
insert kXL 21
insert Njr 19
insert gtT 93
insert CaW 63
find uBW
find gtT
find huE
insert kHU 61
find o
find Ohr
insert oHW 57
find Q
insert VBV 56
insert dQm 3
find RDK
insert oJQ 83
find nAK
insert CDK 6
insert erP 2
find E
find FtW
insert OSt 73
insert ZVy 91
find HkZ
insert Dnz 63
find y
find CaW
insert Dnz 63
find huT
insert NIk 15
insert ple 62
insert DjX 45
insert xLn 59
find OGg
find HkZ
insert BJw 50
insert dQL 71
insert EEN 19
find oJQ
insert FtW 76
insert leb 14
find X
find CaW
find h
insert gwu 76
insert gBe 97
insert BsG 80
insert tYZ 31
find BJw
insert IWC 93
find eWS
find ESn
insert QlO 19
find Fbd
find wkz
insert atc 7
insert VBV 40
insert vQG 69
insert nAK 33
insert oCY 82
insert gtT 93
find CaW
insert oHW 66
find FtW
find FtW
insert RDK 42
insert CaW 45
find RDK
insert hup 36
find huE
insert erP 6
find Bza
find BcN
insert CaW 13
insert fMG 94
insert OGg 63
insert kHU 61
insert QVK 78
find CaW
insert AwM 84
insert ZVy 35
find v
find hhu
insert iCA 90
find huE
insert OGg 63
insert zct 89
insert vUF 29
find U
find F
insert FtW 40
insert tVA 97
insert kMD 7
insert FtW 54
insert BcN 69
insert ctf 86
insert aip 38
insert Dzn 20
insert CaW 63
insert fZA 22
insert kiP 62
find O
find BcN
insert gMZ 57
find CaW
find fkX
insert ZYf 85
insert QSJ 14
find qWH
find CaW
find ctf
insert Dnz 63
insert Eem 67
find DSo
find gwu
insert yyo 80
find gwu
insert dqX 69
insert CaW 89
insert lWH 23
insert BcN 95
insert Yyd 39
insert BrN 73
find Fly
insert tYZ 31
insert pXH 4
find lTm
insert sZJ 0
find sha
find H
find PtO
insert gbG 60
insert tYZ 31
insert sbo 17
insert zbg 40
insert AnO 12
find AMj
find tRQ
find uoi
insert bOO 34
find Ohr
find FtW
find ZYf
insert Sir 57
find Fly
insert nAK 33
insert vQG 70
insert Ixg 8
find BJw
insert Wct 63
insert MGq 59
insert upE 56
find U
insert yLi 57
insert lWH 44
insert qnP 69
insert BcN 49
find U
insert lWH 78
find vQG
insert cOv 25
insert VHd 76
insert lWH 44
find lQv
insert Fly 46
insert BcN 49
insert HGL 68
insert sfz 70
find d
insert VBV 56
insert haL 35
insert VBV 56
insert VBV 89
find zct
insert REn 98
insert VBV 14
find bfU
insert VBV 56
find VBV
insert nAK 33
insert tRG 82
insert Aej 41
insert gtT 93
insert MGq 59
find uoi
insert CaW 67
insert wAe 30
insert Szt 82
insert VBV 32
insert fUR 39
insert xpf 78
insert sha 7
insert sJn 18
insert nAK 33
insert BJw 21
insert lpe 75
find dqX
insert iwt 5
insert Fiq 10
insert BcN 6
insert ObA 4
insert gsf 37
find Eem
insert MGq 59
find CaW
find rtf
insert nAK 33
find BFd
insert eds 5
insert AMj 92
insert gFE 4
insert Njr 19
insert qpQ 84
insert CMa 92
find uoi
insert UMV 66
insert sJn 36
insert gsf 46
insert sFv 59
find akm
find K
find VHd
find VBV
insert HPV 26
insert Sib 32
find G
find y
find b
insert qFN 48
find myG
find CaW
find fMG
find huE
find erP
insert uSp 20
insert cWo 40
insert kUC 14
find Fnm
insert QfS 98
find j
find CaW
insert BQz 94
find gZR
insert Rjn 95
insert bYQ 67
find AMj
find CaW
insert twH 22
insert Tta 70
insert gwu 75
find sha
find huE
insert fsd 79
find gMZ
find Dnz
insert CFR 18
insert oHW 97
insert jDS 54
insert EEN 19
insert nAK 33
insert lWH 78
find sJn
insert CaW 12
insert SAf 14
insert qWH 8
find v
insert rGL 87
insert VMt 71
insert kMD 7
insert Mtw 89
insert VBV 17
find z
insert GZL 50
insert vct 72
find A
find nAK
insert hup 36
insert KxJ 39
find i
find vQG
insert VBV 77
find CaW
insert haL 35
insert CaW 43
find iwt
insert BcN 22
find CaW
find V
insert dQm 3
insert JsB 43
insert CaW 89
insert uBW 48
find gwu
insert Ypx 99
insert JxG 13
insert dDt 32
find FtW
find s
insert CaW 95
find KxJ
find sJn
find W
insert qnP 87
find R
find haL
insert tRQ 2
find FtW
insert ZVy 91
insert BXP 55
insert uSp 20
find seg
insert TgX 74
find Njr
find OEo
find huE
insert XQz 3
insert Sib 32
insert VvQ 82
insert seg 39
insert DSo 47
insert GUL 81
insert FAb 25
find Dnz
insert OLN 68
find CaW
insert jyd 30
insert Nhb 60
find YWd
find gbG
insert ZVy 72
insert ctf 51
find GZa
find CaW
find vQG
insert yqP 98
insert Fiq 10
find HkZ
find zzq
insert ncJ 88
insert WTm 71
find WWF
find K
insert QKG 29
find lWH
insert Vni 3
find e
insert VBV 10
find S
find CaW
insert Xhn 12
insert Sib 32
find U
insert mdw 0
find Sib
insert fbH 42
find uoi
find Fnm
find euc
find BQz
find VBV
insert huT 26
find fkX
insert okZ 61
insert mdw 0
find T
insert gsf 65
insert hQC 97
insert huE 23
insert fpe 85
insert CaW 89
insert ctf 30
find C
insert xnu 43
insert CaW 3
insert qMH 51
find HYE
insert BAM 39
find u
insert wuy 5
find CaW
find ZRp
find fMG
insert Mtw 17
insert bbH 59
insert qWH 8
find dxP
find rtf
insert OdV 21
find X
insert gwu 76
find dDt
find m
insert ctf 51
find euc
find qWH
find BcN
insert ncJ 88
find RDK
insert huT 18
insert Fnm 25
find ctf
insert Dhh 85
insert tst 34
insert lWH 76
find vUF
insert Eem 67
find eJt
insert CaW 67
insert ObA 95
find ple
insert DSo 16
find gwu
insert CaW 89
find BcN
insert huE 76
insert KfY 16
insert XHH 30
insert dQm 3
insert JoU 0
find lWH
insert JlR 6
insert cJa 88
find Fly
find twH
insert VcX 15
insert dQm 3
insert Rku 89
insert rLc 79
insert uYB 82
insert gZb 79
insert nAK 33
insert qoq 84
insert OSt 73
insert jXi 33
find lWH
find akm
