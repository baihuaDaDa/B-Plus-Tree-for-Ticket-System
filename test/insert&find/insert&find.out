null
88 89 
49 
88 89 
null
88 89 
88 89 
48 76 
3 88 89 
3 88 89 
32 
48 76 
49 
3 88 89 
null
3 88 89 
29 49 
11 
3 21 88 89 
3 21 88 89 
3 21 88 89 
32 
null
46 
null
21 
null
78 
29 49 
null
null
null
88 
29 49 
null
null
88 
76 
3 21 51 68 88 89 
3 21 51 68 88 89 
null
23 48 76 
3 21 51 63 68 88 89 
3 21 51 63 68 88 89 
36 
3 21 51 63 68 88 89 
26 48 
3 21 51 63 68 88 89 
46 
null
23 48 76 
76 
3 21 51 63 68 88 89 
29 30 42 49 
57 
29 30 42 49 
60 
3 21 51 63 68 88 89 
3 21 51 63 68 88 89 
null
null
3 21 51 63 68 88 89 
null
null
63 
29 30 42 49 87 
3 21 51 63 68 88 89 
88 
null
23 48 76 
null
46 
46 
3 21 51 63 68 88 89 
3 21 51 63 68 88 89 
31 
null
23 48 76 
null
31 
null
88 
3 21 51 63 68 88 89 
29 30 42 49 87 
63 
15 40 76 
85 
24 32 46 
3 15 21 51 63 68 88 89 
57 
3 15 21 45 51 63 68 88 89 
83 
23 48 76 
6 
3 15 21 45 51 63 68 88 89 
85 
21 
29 30 42 49 69 87 
57 
null
3 15 21 45 51 63 68 88 89 
null
23 48 76 
null
36 45 
null
31 
23 48 76 
44 79 
23 
null
null
23 48 76 
3 15 21 45 51 63 68 88 89 93 
46 
23 44 85 
null
21 
23 44 85 
24 
5 
19 
null
3 15 21 45 51 63 68 88 89 93 
null
6 29 30 42 49 69 87 
88 
null
99 
null
14 24 32 40 46 
3 15 21 45 51 63 68 88 89 93 
null
null
null
70 
11 76 
23 25 48 76 
45 
42 65 
26 
57 
null
45 51 
14 
8 44 79 
88 
null
24 38 
3 15 21 45 51 63 67 68 88 89 93 
3 15 21 45 51 63 67 68 88 89 93 
35 
6 29 30 42 49 69 87 
null
3 15 21 45 51 63 67 68 88 89 93 
3 15 21 45 51 63 67 68 79 88 89 93 
26 
null
null
57 59 
null
3 15 21 45 51 63 67 68 79 88 89 93 
null
93 
78 
3 15 21 45 51 63 67 68 79 88 89 93 
6 81 98 
null
33 
3 15 21 45 51 63 67 68 79 88 89 93 
28 36 
23 25 48 76 
23 25 48 76 
3 15 21 45 51 63 67 68 79 88 89 93 
23 25 48 76 
23 44 85 
28 36 
27 46 80 
27 68 
49 78 
60 
3 15 21 45 51 63 67 68 79 88 89 93 
null
21 
3 15 21 45 51 63 67 68 79 88 89 93 
93 
6 29 30 42 49 69 87 
24 
42 65 
21 
47 
88 
15 36 40 76 
82 
null
14 24 32 40 46 
null
3 15 21 45 51 63 67 68 79 88 89 93 
3 15 21 45 51 63 67 68 79 88 89 93 
3 15 21 45 51 63 67 68 79 88 89 93 
45 
12 
18 77 
8 
21 
36 
75 
null
26 48 
null
3 15 21 45 51 63 67 68 79 88 89 93 
11 76 
27 46 80 
6 29 30 42 49 69 87 
null
57 94 
14 24 32 40 46 
23 44 85 
23 25 48 76 91 
3 15 21 45 51 63 67 68 79 88 89 93 
null
null
27 46 80 
63 
null
3 15 21 45 51 63 67 68 79 88 89 93 
18 52 77 
93 
23 25 48 76 91 
null
85 
null
42 65 
33 
null
7 15 36 40 76 
57 94 
null
3 15 21 45 51 63 67 68 79 88 89 93 
26 
63 
57 94 
83 
null
3 15 21 45 51 63 67 68 79 88 89 93 
null
21 50 
26 
36 
13 45 51 
57 
3 15 21 45 51 63 67 68 79 88 89 93 
7 15 36 40 76 
7 15 36 40 76 
42 65 
23 25 48 76 91 
78 
6 29 30 42 49 69 87 
3 13 15 21 45 51 63 67 68 79 88 89 93 
null
47 
23 25 48 76 91 
null
null
null
6 29 30 42 49 69 87 
3 13 15 21 45 51 63 67 68 79 88 89 93 
14 
8 
3 13 15 21 45 51 63 67 68 79 88 89 93 
6 81 86 98 
47 48 
11 76 
11 76 
27 46 80 
6 
93 
null
14 75 
92 
2 66 
8 44 79 
85 
7 15 36 40 54 76 
85 
27 46 80 
21 50 
null
null
9 14 69 70 
57 
null
89 
89 
14 24 32 40 46 56 89 
8 44 79 
69 
67 
3 13 15 21 45 51 63 67 68 79 88 89 93 
60 
26 
8 44 79 
90 
null
76 
14 24 32 40 46 56 89 
null
null
null
88 
3 13 15 21 45 51 63 67 68 79 88 89 93 
94 
23 25 48 76 91 
2 6 
26 48 
null
3 13 15 21 45 51 63 67 68 79 88 89 93 
36 
92 
3 13 15 21 45 51 63 67 68 79 88 89 93 
7 93 
23 25 48 76 91 
57 
63 
18 36 
null
null
null
33 
null
9 14 69 70 
3 12 13 15 21 45 51 63 67 68 79 88 89 93 
5 
3 12 13 15 21 43 45 51 63 67 68 79 88 89 93 
null
11 75 76 
7 15 36 40 54 76 
null
39 86 
18 36 
null
null
35 65 
7 15 36 40 54 76 
39 
19 
47 
23 25 48 76 91 
63 
3 12 13 15 21 43 45 51 63 67 68 79 88 89 93 95 
38 
60 
42 
3 12 13 15 21 43 45 51 63 67 68 79 88 89 93 95 
9 14 69 70 
57 94 
8 
4 
null
23 44 78 85 
null
null
3 12 13 15 21 43 45 51 63 67 68 79 88 89 93 95 
null
32 
8 44 79 
26 48 
24 38 
94 
10 14 17 24 32 40 46 56 77 89 
14 
null
null
14 
null
3 12 13 15 21 43 45 51 63 67 68 79 88 89 93 95 
28 
94 
41 
60 
null
32 75 
null
24 38 
8 
6 22 29 30 42 49 69 87 95 
42 65 
6 30 51 81 86 98 
29 
92 
62 
11 75 76 
6 22 29 30 42 49 69 87 95 
23 44 76 78 85 
27 46 80 
22 
23 44 76 78 85 
90 
